### PR TITLE
ceph-ansible-pull-requests: force ansible version

### PR DESCRIPTION
**Pull request by leseb**
Ansible 2.2 seems to be breaking things, let's stick with 2.1 for now.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by ktdreyer** [comment]
What was the exact error text with Ansible 2.2? The Ansible engineering team at Red Hat is asking us about the feasibility of moving to 2.2.

**Comment by alfredodeza** [comment]
@ktdreyer 

    TASK [ceph.ceph-common : apply operating system tuning] ************************
    task path: /home/jenkins-build/build/workspace/ceph-ansible-pull-requests/DIST/ceph_ansible_pr_trusty/ceph-ansible/roles/ceph-common/tasks/misc/system_tuning.yml:24
    fatal: [localhost]: FAILED! => {
        "failed": true, 
        "msg": "the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'unicode object' has no attribute 'name'\n\nThe error appears to have been in '/home/jenkins-build/build/workspace/ceph-ansible-pull-requests/DIST/ceph_ansible_pr_trusty/ceph-ansible/roles/ceph-common/tasks/misc/system_tuning.yml': line 24, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: apply operating system tuning\n  ^ here\n"
    }
    	to retry, use: --limit @/home/jenkins-build/build/workspace/ceph-ansible-pull-requests/DIST/ceph_ansible_pr_trusty/ceph-ansible/test.retry


Basically this:

    the field 'args' has an invalid value, which appears to include a variable that is undefined

This wasn't a fatal error before. Ansible has introduced backwards incompatible changes in 2.2 (from 2.1 even)

**Comment by samdoran** [comment]
@leseb @alfredodeza  This error is due to the deprecation of bare variables in loops, which has been [warning since Ansible 2.0](https://docs.ansible.com/ansible/porting_guide_2.0.html#deprecated).

Deprecated:

```
with_items: os_tuning_params
```

Correct syntax:

```
with_items: "{{ os_turing_params }}"
```

From looking through [ansible-ceph-common](https://github.com/ceph/ansible-ceph-common), I see three instances of this. 

```
ansible-ceph-common/tasks/installs/install_rh_storage_on_debian.yml:
   38      update_cache: yes
   39      cache_valid_time: 3600
   40:   with_items: debian_package_dependencies
   41  
   42  - name: install red hat storage ceph mon

ansible-ceph-common/tasks/installs/install_rh_storage_on_redhat.yml:
   21      name: "{{ item }}"
   22      state: present
   23:   with_items: redhat_package_dependencies
   24    when:
   25      - ansible_pkg_mgr == "yum"

ansible-ceph-common/tasks/misc/system_tuning.yml:
   29      sysctl_file: /etc/sysctl.conf
   30      ignoreerrors: yes
   31:   with_items: os_tuning_params
   32  
```

This repo has 18 instances.

**Comment by leseb** [comment]
Thanks @samdoran, we will address those issues shortly.
